approval_polls/views.py

```diff
@@ -12,6 +12,13 @@
 
 def index(request):
   poll_list = Poll.objects.filter(pub_date__lte=timezone.now()).order_by('-pub_date')
+  return GetPolls(request, poll_list, 'approval_polls/index.html')
+
+def PollsView(request):
+  poll_list = Poll.objects.filter(pub_date__lte=timezone.now(), user_id=request.user).order_by('-pub_date')
+  return GetPolls(request, poll_list, 'approval_polls/my_polls.html')
+
+def GetPolls(request, poll_list, render_page):
```

Comment:
For code consistency reasons `GetPolls` should be `getPolls` (method names start with a lowercase letter and class names start with an uppercase letter).